src/main/app/forms/models/documentType.ts

```diff
@@ -0,0 +1,50 @@
+import { MaxLength, ValidateIf } from 'class-validator'
+import { Serializable } from 'models/serializable'
+
+export class ValidationErrors {
+  static readonly TEXT_TOO_LONG: string = 'You’ve entered too many characters'
+}
+
+export class DocumentType implements Serializable<DocumentType> {
+
+  particularsOfClaim?: boolean
+  responsePack?: boolean
+  scheduleOfLoss?: boolean
+  medicalReport?: boolean
+  other?: boolean
+
+  @ValidateIf(o => o.other === true)
+  @MaxLength( 255, { message: ValidationErrors.TEXT_TOO_LONG } )
```

Comment:
Why not `ValidatorConstants` class to store all consts (usefull for unit tests).
Please see `features/response/form/models/evidenceRow/ValidationConstants`